### ISSUE TITLE
Syntax error executing simple aggregate

### ISSUE THREAD

**Issue by craigh1015** [Microsoft/vscode-cosmosdb#688]
Repro steps:

Run following mongo query from connected scrapbook
```javascript
db.hdr.aggregate([
  { $match: { "CURRENCY_ID": "USD" } },
])
```
The same query runs fin from the mongo client.

Action: cosmosDB.executeMongoCommand
Error type: Error
Error Message: Error near line 0, column 19, text '$match'. Please check syntax.

Version: 0.7.2
OS: darwin

**Comment by PrashanthCorp** [comment]
Hi Craig, 
Thanks for filing a bug report! Your report helps us prioritize what to work on. 
We're working on better error reporting. I see two places our code would complain about: 
1. Quotes around `$match`. See #466 and #467. 
2. Trailing comma inside the array. 
I tried a similar query (below) and it seems to execute.  
```
db.<collectionName>.aggregate([
  { "$match": { "CURRENCY_ID": "USD" } }
])
```

Let me know if this works.

**Comment by craigh1015** [comment]
Yes this works - thanks

**Comment by PrashanthCorp** [comment]
Hi Craig, the merged fix was published today (version 0.8.0). Hope this fixes your issue. :-)
Let us know how the fix holds up with your use-case.
